src/executables/util_cs.c

```diff
@@ -4040,9 +4040,11 @@ flashback (UTIL_FUNCTION_ARG * arg)
   char table_name_buf[SM_MAX_IDENTIFIER_LENGTH];
   char *table_name;
 
+  OID *oid_list = NULL;
```

Comment:
I would like to give you one opinion. In the next PR, how about combining the variables used in the flashback operation into a single context?

Reply:
Ok, that will be more readable when pass these variables as arguments. 
I will follow it. 
In this PR, these variables are used for testing.